conda-recipe/run_test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# if dpc++ vars path is specified
+if [ ! -z "${DPCPP_VAR}" ]; then
+    source ${DPCPP_VAR}
+fi
+
+# if DAALROOT is specified
+if [ ! -z "${DAALROOT}" ]; then
+    conda remove daal --force -y
+    source ${DAALROOT}/env/vars.sh
+fi
```

Comment:
What is this script supposed to do?

Reply:
This script runs at testing stage like build.sh runs at building stage. It enables DPC++ compiler runtime (paths) and replace daal from conda environment (package) with daal from DAALROOT. (if these env vars were specified and were used during the build stage)

Reply:
I don't get it. This script only sets env-vars. How are the tests run?

Reply:
Tests run as usual by commands in the test section in the meta.yaml file.